app/views/items/_item_largescreen.html.erb

```diff
@@ -63,7 +63,7 @@
           <%= link_to I18n.t("items.buttons.back"), items_path %>
           <%# <%= button_to "Destroy this item", @item, method: :delete %>
           <div class="d-flex justify-content-center w-100 mb-5">
-            <%= if current_user.lending_rights?(@item)
+            <%= if current_user.lending_rights?(@item) && @src_is_qrcode
```

Comment:
Könnt ihr hier anstatt dem `&&` ein `and` nutzen? Zumindest in unserer Codebase wurde bisher recht konsistent die ausgeschriebene Variante benutzt.